app/src/main/java/com/kylecorry/trail_sense/tools/tools/ui/Tools.kt

```diff
@@ -539,7 +539,7 @@ object Tools {
             .flatMap { it.quickActions }
             .distinctBy { it.id }
             .sortedBy { it.id }
-            .map { it.copy(name = it.name.capitalizeWords()) }
+            .map { it.copy(name = it.name) }
```

Comment:
Can you undo this change? I don't want the tool quick action names to be changed at this point in time